File: app/src/main/java/com/future/frouterdemo/router/java/TestRouter.java
package com.future.frouterdemo.router.java;

import com.future.frouter.annotation.FRounter;
import com.future.frouterdemo.router.kotlin.ITestRouter;

@FRounter(module = "哈哈哈")
public class TestRouter implements ITestRouter {
    @Override
    public String test() {
        System.out.printf("测试啊");
        return "哈哈哈";
    }

}

File: app/src/main/java/com/future/frouterdemo/router/kotlin/TestRouter.kt
package com.future.frouterdemo.router.kotlin

import com.future.frouter.annotation.FRounter

@FRounter("app")
class TestRouter: ITestRouter {
    override fun test(): String {
        println("哈哈哈")

        return "哈哈哈";
    }
}